### DISCUSSION TITLE
History of aio-libs

### DISCUSSION THREAD

**Discussion by dereckt** [aio-libs/.github#22] [Q&A] [answered]
Hi,

I made a few pull requests earlier to yarl as per part of a University project I'm currently working on. I'm unsure as to how often people check pull requests on this project and not sure where to contact people for help/advice. Would anyone be willing to review these pull requests. Also, I'm curious on aio-libs and its history. Is there some documentation I can read or someone who'd be willing to briefly talk to me about it?

Thanks,
-Dereck

**Comment by webknjaz** [comment, marked as answer]
The maintainers look at the PRs according to their time availability, so it depends. Among other things, there's a Matrix chat at https://matrix.to/#/%23aio-libs:matrix.org (and an umbrella space at https://matrix.to/#/%23aio-libs-space:matrix.org). The org was created in parallel with the creation of the first implementations of asyncio, maybe a bit earlier than it got into stdlib.
